### ISSUE TITLE
Error message: Multi-dimensional coordinate not supported: 'lataitude', should be "unable to find FX file"

### ISSUE THREAD

**Issue by ledm** [ESMValGroup/ESMValCore#1679]
From this question in discussions:

### Discussed in https://github.com/ESMValGroup/ESMValTool/discussions/2746

<div type='discussions-op-text'>

<sup>Originally posted by **RChapman1407** July 26, 2022</sup>
Hello,

I am having difficulty getting one of the ocean diagnostic scripts to work with CMIP6. So far it works fine with CMIP5. The recipe is recipe_ocean_example.yml. I am trying to average some ocean salinities over defined volumes before extracting them to use in a conceptual model. 
Should I be giving a different route to the datasets? I am currently using the same structure of path as the predefined datasets, and replacing the details for the data I want. I am working within JASMIN.
I have already tried giving the grid as well, but this did not work. I currently get the error 'Failed to run preprocessor function 'volume_statistics' on the data' and in the trace the comment 'Multi-dimensional coordinate not supported: 'lataitude''

Thanks</div>

Basically, the error 'Failed to run preprocessor function 'volume_statistics' on the data' and in the trace the comment 'Multi-dimensional coordinate not supported: 'lataitude'

means that you need FX files, but the preprocessor can't find it. Can we re-write this error message so that it is clearer?

Cheers!
